.gitlab-ci/packet.yml

```diff
@@ -148,9 +148,10 @@ packet_centos7-calico-ha-once-localhost:
   when: on_success
   variables:
     # This will instruct Docker not to start over TLS.
+    DOCKER_LOCALHOST_ENABLE: "true"
```

Comment:
We have `docker:26.1.4-dind` in the service. (GitLab CI)
I'm wondering if we just need to add the `DOCKER_HOST` variable so that we don't need `service docker start`.

```suggestion
    DOCKER_HOST: tcp://docker:2375
```

Reply:
Let me try

Reply:
Thanks @tico88612 , it works.